src/Core/Silk.NET.Core/Miscellaneous/RawImage.cs

```diff
@@ -0,0 +1,87 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Linq;
+
+namespace Silk.NET.Core
+{
+    /// <summary>
+    /// Represents loaded, uncompressed, processed image data.
+    /// </summary>
+    public readonly struct RawImage : IEquatable<RawImage>
+    {
+        /// <summary>
+        /// Creates a <see cref="RawImage"/> given pixel data and pixel dimensions.
+        /// </summary>
+        /// <param name="width">The width of the image.</param>
+        /// <param name="height">The height of the image.</param>
+        /// <param name="rgbaPixels">The image daqta.</param>
+        public RawImage(int width, int height, byte[] rgbaPixels)
```

Comment:
could also take `Memory` here.
```suggestion
        public RawImage(int width, int height, Memory<byte> rgbaPixels)
```